plugins/processors/ifname/ifname.go

```diff
@@ -349,6 +349,7 @@ func init() {
 func makeTableNoMock(fieldName string) (*si.Table, error) {
 	var err error
 	tab := si.Table{
+		Name:       "ifName",
```

Comment:
```suggestion
		Name:       "ifTable",
```

Looks good to me, just would pick a more correct name..

Reply:
How is ifTable more correct? Especially since it is the ifName processor.

Reply:
ifTable is the snmp table where you read the interface name